apps/etterlatte-beregning/src/main/kotlin/avkorting/AvkortingValider.kt

```diff
@@ -1,33 +1,59 @@
 package no.nav.etterlatte.avkorting
 
-import no.nav.etterlatte.libs.common.behandling.BehandlingType
-import no.nav.etterlatte.libs.common.behandling.DetaljertBehandling
-import no.nav.etterlatte.libs.common.behandling.virkningstidspunkt
 import no.nav.etterlatte.libs.common.beregning.AvkortingGrunnlagLagreDto
 import no.nav.etterlatte.libs.common.feilhaandtering.IkkeTillattException
 import java.time.Month
+import java.time.YearMonth
 
 object AvkortingValider {
     fun validerInntekt(
         nyInntekt: AvkortingGrunnlagLagreDto,
         avkorting: Avkorting,
-        behandling: DetaljertBehandling,
+        innvilgelse: Boolean,
     ) {
-        skalIkkeKunneLeggeTilEllerEndreAarsinntektTidligereEnnForrigeAarsinntekt(behandling, avkorting)
-        skalIkkeLeggeTilFratrekkInnAarHvisDetErEtFulltaar(avkorting, nyInntekt, behandling)
+        foersteRevurderingIAareneEtterInnvilgelsesaarMaaStarteIJanuar(
+            nyInntekt,
+            avkorting,
+            innvilgelse,
+        )
+        skalIkkeKunneLeggeTilEllerEndreAarsinntektTidligereEnnForrigeAarsinntekt(
+            nyInntekt.fom,
+            avkorting,
+        )
```

Comment:
Skal det være mulig å endre fjorårets inntekt? Det vil jo ikke slå ut i noen restanse-innhenting, siden vi har ingen måneder å hente inn over -- eller vil vi støtte dette for å kunne få en tidligere tilbakekreving / etterbetaling enn etteroppjøret?

Reply:
Det vi bestemt så langt er at endring av inntekt kun kan skje fremover, altså virk er måned etter inntekt er rapportert. Eneste tilfelle hvor vi endrer tilbake i tid så er det hvis det er blitt lagt til feil av saksbehandler eller lignende. Og da vil det medføre tilbakekreving/etterbetaling.